tests/integ/modin/test_unimplemented.py

```diff
@@ -189,13 +189,7 @@ def test_unsupported_str_methods(func, func_name, caplog) -> None:
     lambda idx: idx.factorize(),
     lambda idx: idx.insert(),
     lambda idx: idx.is_(),
-    lambda idx: idx.is_boolean(),
     lambda idx: idx.is_categorical(),
-    lambda idx: idx.is_floating(),
-    lambda idx: idx.is_integer(),
-    lambda idx: idx.is_interval(),
```

Comment:
should still keep is_interval here, even if it's not on the roadmap to implement